aws/cloudflare-lb-pools/example/main.tf

```diff
@@ -0,0 +1,27 @@
+module "cloudflare_load_balancer_pool" {
```

Comment:
@Mshahidtaj Nice thought to add example. We might consider it in other places too.

Reply:
I have added in other places too @stafot not everywhere though